terraform.tfvars.example

```diff
@@ -42,10 +42,10 @@ control_plane_count = 3
 compute_count = 2
 
 // Ignition config path for the control plane machines
-control_plane_ignition_path = "./master.ign"
+control_plane_ignition_path = "./master.ign" //TODO
 
 // Ignition config path for the compute machines
-compute_ignition_path = "./worker.ign"
+compute_ignition_path = "./worker.ign" //TODO
 
 // The IP address to assign to the load balancer VM.
```

Comment:
Are the same IP addresses able to be used here?

Reply:
Yes, i believe so.